.eslintrc.json

```diff
@@ -116,7 +116,7 @@
 		"es6": true
 	},
 	"parserOptions": {
-		"ecmaVersion": 2018,
+		"ecmaVersion": 2021,
```

Comment:
Hm... What about all the services services/modules not compatible with this? I notice `debitoor-app` is still running [on node 12](https://github.com/debitoor/debitoor-app/blob/master/package.json#L218) for example. Wouldn't that cause issues and make it incompatible?

Reply:
yep, indeed (
no I'm think to change only in `"ecmaVersion": 2021,` only api (

Reply:
I have previously seen projects where they managed to create multiple configs in their eslint configuration. Would be nice to do something similar here, if possible :)

Reply:
Accounting/integrations/platform has many services where I believe `2021` would make sense too.

Reply:
something like this https://github.com/debitoor/eslint-config-api ? )
quite strange to use https://github.com/debitoor/debitoor-api/blob/0f92f29f7aa3e1291e481ee1f7d0690d7ef11044/package.json#L166-L167

Reply:
Hm, no. Not exactly. So you just have one package and on usage, when extending, you define a specific config, unless you are using default. Imagine `.eslintrc.yml` files:

Default:

```yaml
extends: "@debitoor/eslint-config-debitoor"
```

Use "deprecated", old version, ecma specific version or node version :

```yaml
extends: "@debitoor/eslint-config-debitoor/es-2015"
```

I will try to remember where I saw it or if I can find a working example somewhere :)

Reply:
this is more than enough )

Reply:
They seem to be doing something like it in airbnb but their solution do not seem simple :-D